dockerfiles/lib/src/api/wsmaster/system/system.ts

```diff
@@ -15,7 +15,7 @@ import {Websocket} from "../../../spi/websocket/websocket";
 import {HttpJsonRequest} from "../../../spi/http/default-http-json-request";
 import {DefaultHttpJsonRequest} from "../../../spi/http/default-http-json-request";
 import {HttpJsonResponse} from "../../../spi/http/default-http-json-request";
-import {MessageBus} from "../../../spi/websocket/messagebus";
+import {JsonRpcBus} from "../../../spi/websocket/json-rpc-bus";
```

Comment:
now, we can get rid of MessageBus class as well and all references to it.

Reply:
ok